components/03-organisms/menu/primary-nav/_yds-primary-nav.scss

```diff
@@ -40,6 +40,18 @@
       z-index: 1;
       border-color: var(--color-navigation-border);
       border-bottom-color: var(--color-background);
+
+      .primary-nav__item--level-0:first-child & {
+        padding-left: var(--size-spacing-5);
+      }
```

Comment:
Good call on this! It's too bad that it makes it "jump" when you expand and collapse it... but I don't know what we can do to avoid it.

We should also add the padding-right to the last item, in case it's a toggle, instead of a link.